internal/core/metadata/v2/controller/http/deviceprofile.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/edgexfoundry/go-mod-bootstrap/di"
 	"github.com/edgexfoundry/go-mod-core-contracts/clients"
 	"github.com/edgexfoundry/go-mod-core-contracts/errors"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2"
```

Comment:
line 18 and 19 are the same.  please remove line 19

Reply:
Removed Line 19.